rustls/src/lib.rs

```diff
@@ -108,9 +108,11 @@
 //! This is the minimum you need to do to make a TLS client connection.
 //!
 //! First we load some root certificates.  These are used to authenticate the server.
-//! The recommended way is to depend on the `webpki_roots` crate which contains
+//! The recommended way is to depend on the [`webpki_roots`] crate which contains
```

Comment:
Actually, this also isn't currently really the recommended way?

Reply:
I think there isn't a single recommended way -- reading https://github.com/rustls/rustls-native-certs#should-i-use-this-or-webpki-roots and making a decision based on that is what I'd advocate (prior to rustls-native-verifier, at least!)

Maybe reword this to "The simplest way is to ..."?

Reply:
Switched to describe as "the simplest way" and to link to the webpki-roots crate.